web-client/public/js/constants.js

```diff
@@ -0,0 +1,2 @@
+export const GREY_EDGES_DASHED_MENU = "#grey-edges-dashed-menu";
+export const GREY_EDGES_DASHED_SIDEBAR = "#dashedGrayLineButton";
```

Comment:
Consolidation into a `constants` module was not initially called for but the need for it was noticed right away based on the way these values were used. Another option instead of a separate `constants` file is to also fold these into `grnState`. Either way could work.